packages/elements/src/components/ino-dialog/ino-dialog.scss

```diff
@@ -36,55 +36,63 @@ $material-dialog-box-shadow: 0px 0px 22px rgba(93, 91, 105, 0.35);
   @include dialog.container-fill-color($background-color);
   @include dialog.scrim-color(theme.$p-4, 0.25);
 
-  .mdc-dialog__scrim {
-    backdrop-filter: blur(5px);
-  }
-
-  .mdc-dialog .mdc-dialog__container {
-    height: #{$height};
-    min-height: #{$min-height};
-    max-height: #{$max-height};
-    width: #{$width};
-    max-width: #{$max-width};
-    min-width: #{$min-width};
-  }
-
-  .mdc-dialog .mdc-dialog__surface {
-    overflow: visible;
-    min-height: unset;
-    max-height: unset;
-    height: 100%;
-    width: 100%;
-    border-radius: $border-radius;
-    box-shadow: $box-shadow;
-    border: #{$border};
-    padding: #{$padding};
-  }
-
-  .mdc-dialog .mdc-dialog__surface header {
-    display: flex;
-    align-items: center;
-    gap: 2rem;
-
-    ino-icon {
-      --icon-width: 2rem;
-      --icon-height: 2rem;
+  .mdc-dialog {
+    &__scrim {
+      backdrop-filter: blur(5px);
     }
-  }
-
-  .mdc-dialog .mdc-dialog__surface header h1 {
-    margin: 0 0 0;
-    font-weight: 400;
-  }
 
-  .mdc-dialog .mdc-dialog__surface section {
-    margin: 38px 0 0;
-  }
+    &__container {
+      height: $height;
```

Comment:
Oh ok, I wasn't aware of that problem. But as far as i understand the referenced article, it shouldn't be a problem in this scenario because the sass variable is not used in any css variable (the exposed variable is --ino-dialog-height). Or am I wrong? I've tested it as well and everything works as expected.